src/ProtectedRoomsSet.ts

```diff
@@ -205,51 +202,33 @@ export class ProtectedRoomsSet {
         }
         this.protectedRoomActivityTracker.handleEvent(roomId, event);
         if (event['type'] === 'm.room.power_levels' && event['state_key'] === '') {
-            // power levels were updated - recheck permissions
-            this.errorCache.resetError(roomId, ERROR_KIND_PERMISSION);
             await this.managementRoomOutput.logMessage(LogLevel.DEBUG, "Mjolnir", `Power levels changed in ${roomId} - checking permissions...`, roomId);
             const errors = await this.protectionManager.verifyPermissionsIn(roomId);
-            const hadErrors = await this.printActionResult(errors);
-            if (!hadErrors) {
-                await this.managementRoomOutput.logMessage(LogLevel.DEBUG, "Mjolnir", `All permissions look OK.`);
-            }
+            await this.printActionResult(errors, { title: "Problem verifying permissions.", noErrorsText: "All permissions look OK."});
             return;
         } else if (event['type'] === "m.room.member") {
             // The reason we have to apply bans on each member change is because
             // we cannot eagerly ban users (that is to ban them when they have never been a member)
             // as they can be force joined to a room they might not have known existed.
             // Only apply bans and then redactions in the room we are currently looking at.
-            const banErrors = await this.applyUserBans([roomId]);
-            const redactionErrors = await this.processRedactionQueue(roomId);
-            await this.printActionResult(banErrors);
-            await this.printActionResult(redactionErrors);
+            const errors = [
+                await this.applyUserBans([roomId]),
+                await this.processRedactionQueue(roomId),
+            ].flat();
+            await this.printActionResult(errors, { title: 'Problem updating member bans.' });
```

Comment:
`printActionResult` didn't used to display a message when there were no errors so this might be a problem here